android/build.gradle

```diff
@@ -51,7 +51,7 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':capacitor-android')
-    implementation 'io.sentry:sentry-android:4.3.0'
+    implementation 'io.sentry:sentry-android:5.0.0'
     implementation 'com.google.code.gson:gson:2.8.6'
```

Comment:
just noticed this, sentry-android already bundles `gson:gson`, why do we have its dependency here? I'd remove it and let sentry-android handle it

Reply:
Hmm, it was used here to iterate through breadcrumb data: https://github.com/getsentry/sentry-capacitor/blob/fef9ff043778740fc8f54883f2cff05652ce3659/android/src/main/java/io/sentry/capacitor/SentryCapacitor.java#L290-L300

But we don't need it as we can just iterate through the JSONObject like we do on the other SDKs...let's refactor it

https://github.com/getsentry/sentry-cordova/blob/2d3efdc8bb0fb038b8cc3d60c2838e0e9a22973f/src/android/io/sentry/SentryCordova.java#L343-L352